flocker/control/test/test_persistence.py

```diff
@@ -363,6 +415,13 @@ def test_sequential_migrations(self):
 
 DATASETS = st.builds(Dataset, dataset_id=UUIDS, maximum_size=st.integers())
 
+LEASES = st.builds(
+    Lease, dataset_id=UUIDS, node_id=UUIDS,
+    expiration=st.one_of(
+        st.none(),
+        st.basic(generate=lambda r, _: datetime.fromtimestamp(
+            r.randrange(0, 10000000), tz=UTC))))
```

Comment:
I suspect `integers(max_value=10000000).map(lambda t: datetime.fromtimestamp(t, tz=UTC))` would perform better. It's cleaner too.